pytorch_lightning/tuner/auto_gpu_select.py

```diff
@@ -15,6 +15,13 @@
 
 
 def pick_multiple_gpus(nb):
+    if nb == 0:
+        ResourceWarning(
```

Comment:
Maybe replace this with a `MisconfigurationException`? 

Message: `auto_select_gpus=True, gpus=0` is not a valid configuration. Please select a valid number of GPU resources when using `auto_select_gpus`.

Reply:
`test_pick_multiple_gpus` will have to reflect this change as well